variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/filter/FilterBuilder.java

```diff
@@ -43,6 +44,18 @@
                    .build();
     }
 
+    public List<VariantRepositoryFilter> getBeaconFilters(Long start,Long startMin,Long startMax,Long end,Long endMin,Long endMax,String referenceBases,String alternateBases,VariantType variantType,List<String> studies){
```

Comment:
Please try to adhere to the existing coding style:

* Lines shouldn't be longer than 120 characters
* A comma must be followed by a whitespace, but in this case it should be a newline (see the previous method)
* A blank line should follow the end of a method to increase readability
* A whitespace surrounding operators like == or !=
* A whitespace before curly braces

Reply:
I have fixed this and have made changes in other places as well and have tried to stick to the existing coding style :)